src/main/java/TaskList.java

```diff
@@ -0,0 +1,62 @@
+public class TaskList {
+    /** Array of tasks */
+    private Task[] tasks;
+
+    /** Number of tasks */
+    private int noOfTasks = 0;
+    public TaskList(int taskLength) {
+        tasks = new Task[taskLength];
+    }

```

Comment:
Unnecessary step of reassigning variable. Assume that the `taskLength` is fixed at 100 as suggested in the assignment, then you can immediately declare and assign into the variable in line 3